source/shared/smart-contracts/guides/fallback-entrypoints.rst

```diff
@@ -0,0 +1,93 @@
+.. fallback-entrypoints:
+
+====================
+Fallback entrypoints
+====================
+
+This guide explains how to use fallback entrypoints, which can be useful
+when creating a proxy for your smart contract.
+
+Preparation
+===========
+
+Make sure to have Rust and Cargo installed and the ``wasm32-unknown-unknown``
+target, together with ``cargo-concordium`` and the Rust source code for a smart
```

Comment:
This sentence does not read very well. The part about the target seems off, and should be rephrased.